spec/views/reviews/review_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+include Authentication
+
+describe 'reviews', type: :feature do
+  context 'anonymous user' do
```

Comment:
For this test group you can replace `reviews` into before section. It is used in every test.